### PR TITLE
EQM: Difficult questions reports fix

### LINKED ISSUE #12147: Difficult questions are not displayed
_Originally posted by @radinamatic in https://github.com/learningequality/kolibri/issues/12111#issuecomment-2100740029_:

> Difficult questions are not displayed. I had 3 learners give incorrect answers to same 6 questions. 

   difficult-questions

[db-logs.zip](https://github.com/learningequality/kolibri/files/15250374/db-logs.zip)

### LINKED ISSUE #12164: EQM: Coach reports for specific "difficult question" viewing failing
Per @pcenov https://github.com/learningequality/kolibri/pull/12111#issuecomment-2102844230

Likely an issue within the component related to how/where the question's content is being fetched. 

https://private-user-images.githubusercontent.com/79847249/329287803-f0987878-f8f4-47f9-a020-46f162cadc47.mp4?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU2MzgyMjAsIm5iZiI6MTcxNTYzNzkyMCwicGF0aCI6Ii83OTg0NzI0OS8zMjkyODc4MDMtZjA5ODc4NzgtZjhmNC00N2Y5LWEwMjAtNDZmMTYyY2FkYzQ3Lm1wND9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDIyMDUyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJlMmZmNGRhYmNlZGQwZDIwOTdkY2JlYzdlZDI1MzEyZjdmMzk4NmJkNjBmODY4MTEwNjM2OTRhNTBkNGEwY2ImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.hCmbqDFYUxGE14xxJuMqME2vRdf5uvKU-bvkOUzsSAc

### PR DESCRIPTION
**Pull request by nucleogenesis**
## Summary

Closes #12164  - where questions were needed in `questionDetail/handlers.js` we were still expecting the Exam.question_sources v2 schema. I did a cursory search for possible other uses of `question_sources` to derive questions but found nothing else.

#### 🎆 BONUS FIXES 🎆 

**Before:**

_Not all issues shown in screenshot_

- Broken icons
- Text and icon were inline-block, weird text overflow
- `px` values
- icon is a few pixels below centered

image

**After:**

_Not all fixes shown in screenshot_

- Proper icons
- Icon vertical centered
- Text is padded from left, justified against icon's spacing
- Icon absolute positioned
- `em` values

image

---

## Reviewer guidance

- [ ] Can you view Difficult questions reports?
- [ ] Does the page have working icons that look right?

### CODE REVIEW

File: kolibri/plugins/coach/assets/src/views/common/QuestionDetailLearnerList.vue [hunk 1]
```diff
@@ -24,13 +24,13 @@
...
               :style="{ fill: $themeTokens.annotation }"
-              icon="cancel"
+              icon="notStarted"
             />
             <KIcon
               v-else-if="!learner.correct"
               class="item svg-item"
               :style="{ fill: $themeTokens.incorrect }"
-              icon="cancel"
+              icon="incorrect"
```
Comment (nucleogenesis) [inline kolibri/plugins/coach/assets/src/views/common/QuestionDetailLearnerList.vue:33]: This feels like it should be the right thing to use here... not sure why `cancel` was used for both of these icons. `cancel` no longer exists and I _assume_ it's what turned into `incorrect` given the `$themeToken` that was used here.

- `notStarted` is used when the learner has no attempts
- `incorrect` is used when the user answered incorrectly (and presumably needs help)

**Comment by pcenov** [comment]
Hi @nucleogenesis I confirm that now it's possible to see the details of the difficult questions however when I go to Coach > Reports > Lesson > Difficult questions I am not seeing the incorrect answers of the learner:

https://github.com/learningequality/kolibri/assets/79847249/1bef017a-4f36-4fc5-979a-dc8930475e0e

If that's not caused by the changes made here, I can file a follow-up issue.

**Comment by nucleogenesis** [comment]
@pcenov I think that @rtibbles will have the issue you found resolved with his Perseus work

Thanks for giving this a look!

**Comment by rtibbles** [comment]
@pcenov I filed your observation as an issue, and tagged it as resolved by my PR here: https://github.com/learningequality/kolibri/pull/12362 if you could verify the fix there, that would be great!

**Review by rtibbles — APPROVED** [review]
All changes make sense, and manual testing confirms the fix.

**Pull request merged**